docs/USAGE.md

```diff
@@ -43,17 +44,17 @@ Some values within the GlobalLink Connect dashboard are configurable at the poin
 When you create your content, there are a couple of things you will need to do to ensure that this content is available for translation:
 
 1. Assign the source locale to your content item.
-   - For example, if you are looking to translate your content from English to German, you may want to assign en-US, or en-GB.
-2. (Optionally) assign the content item to a person.
-   - This is not mandatory for translation but may help with finding the content item when you create a submission, as content items can be filtered by assignee.
+   - For example, if you are looking to translate your content from English to German, you may want to assign en-US.
+2. (Optionally) assign the content item to a person/s.

```

Comment:
Odd, `a persons` doesn't make much sense. Could replace with "someone" if you want it to be more ambiguous that you can assign more than one person.